### DISCUSSION TITLE
Comprehensive configuration for daily.dev search bar

### DISCUSSION THREAD

**Discussion by burhanalphasquad** [dailydotdev/daily#1591] [Feature Requests]
Hi, community, first of all, I want to acknowledge the service daily.dev provides by giving such a rich integration. And I'm an avid user of this. And since my proper usage, I started to realize that there is something I need a bit too much in my general usage
Here it goes
I would consider myself a proferssional developer who is a power user, and during my work, I want to have focus, which means that whenever I open a new page in broswer, I want the option to have a 
1. Default configuration, which searches (the daily dev searchbar, cuz I use the browser in fullscreen while doing my work) which goes to anywhere where I set it to (Google, ChatGPT, StackOverflow, whatever)
2. A dropdown of options which give me a quick way to deviate from my default configuration as well. 

And if there is a way to do this currently, I would appreciate if someone guides me. 

Regards

**Comment by idoshamun** [comment]
Hi @burhanalphasquad,

Please elaborate your request. I'm not sure I follow exactly what you need
